### ISSUE TITLE
Convert utf  based text file to translate/create mp3 audio for patreon membership ?

### ISSUE THREAD

**Issue by mxav1111** [AwesomeTTS/awesometts-anki-addon#309]
Hi there,
Is there a way we can use this tool to convert utf based text file to translate/create mp3 audio for patreon membership ?
We can provide source language, destination language,  utf encoded text file and it can create (one or more mp3 files) with flexibility of choosing the speed (rate) of speech.

Thanks for the awesome tool.

-Max.

**Comment by luc-languagetools** [comment]
Can you give me more details about what you're trying to do ?

**Comment by luc-languagetools** [comment]
yes unfortunately this would be too drastic of a departure for what AwesomeTTS is trying to do
